managed-databases/postgresql-and-mysql/api-cli/pg-audit.mdx

```diff
@@ -0,0 +1,303 @@
+---
+meta:
+ title: Setting up and using the pgaudit extension
+ description: This page explains how to set up and use the pgaudit extension.
+content:
+ h1: Setting up and using the pgaudit extension
+ paragraph: This page explains how to set up and use the pgaudit extension.
+tags: managed-database postgresql pgaudit pg-extensions
+dates:
+ validation: 2024-03-05
+ posted: 2024-03-05
+categories:
+ - managed-databases
+ - postgresql-and-mysql
+---
+
+The `pgaudit` extension is available with Scaleway Managed Databases for PostgresSQL. The extension is a powerful monitoring and logging tool that allows you to keep track of the actions happening in your databases, and record them.
+
+The main purpose of `pgaudit` is to allow PostgreSQL users to produce audit logs often required to comply with government, financial, and/or ISO certifications.
+
+`pgaudit` allows you to log different statements executed in your Database Instance, or specific databases. You can keep track of who accessed your databases, what actions were performed, and when, under a [structured format](https://github.com/pgaudit/pgaudit/blob/master/README.md#format).
+
+You can configure the extension to log only what is relevant to your audits.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A [Database Instance](/managed-databases/postgresql-and-mysql/how-to/create-a-database) running a PostgreSQL engine
+
+## Enabling and installing pgaudit
+
+1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu of the Scaleway console. A list of your Database Instances displays.
+2. Click the Database Instance name or <Icon name="more" /> > **More info** to access the Instance information page.
+3. Click the **Advanced settings** tab.
+4. Click <Icon name="edit" />.
+5. Click **+ Add parameters**.
+6. Select `rdb.enable_pgaudit` in the drop-down and click the toggle to set it to `on`.
+7. Click <Icon name="validate" /> to validate. Your Database Instance restarts.
+
+   <Message type="important">
+     Restarting can take up to a few dozen seconds depending on the size of your Database Instance.
```

Comment:
maybe something generic than "few dozen"

Reply:
cc @scw-wtimmermans what do you think?

Reply:
I prefer to be specific, personally

Reply:
```suggestion
     Restarting can take several seconds depending on the size of your Database Instance.
```

what about this?